backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/IShipmentScheduleBL.java

```diff
@@ -128,7 +128,7 @@ public interface IShipmentScheduleBL extends ISingletonService
 	 */
 	void openShipmentSchedule(I_M_ShipmentSchedule shipmentScheduleRecord);
 
-	void closeShipmentSchedulesFor(ImmutableList<TableRecordReference> orderLineRecordRefs);
+	void closeShipmentSchedulesFor(ImmutableList<TableRecordReference> orderLineRecordRefs, final boolean isErrorIfProcessed);
```

Comment:
minor: i would suggest calling it isThrowErrorIfAlreadyProcessed 

but that's my taste. if u don't agree with it, pls don't do it.

Reply:
it was introduced in https://github.com/metasfresh/metasfresh/pull/5961 to prevent problems on reactivate in this case. This logic I wanted to preserve with this param.